movieapp/src/components/Navigation/Navigation.tsx

```diff
@@ -11,17 +13,17 @@ const navmenu: Array<Menu> = [
     link: 'all'
   },
   {
-    id: 'all',
+    id: 'all1',
```

Comment:
if this component will get this config as a prop - it will be component)

Reply:
fixed